### PR TITLE
Real-to-Virtual Data Products Documentation

### LINKED ISSUE #173: Document Real-to-Virtual Data Products
As part of the Real-to-Virtual software, monoDrive is providing interim data products from the Real-to-Virtual pipeline. We need a place for the documentation for these data products to live for customers.

### PR DESCRIPTION
**Pull request by bradleynull**
This completes the documentation for the data products in the current Real-to-Virtual pipeline. I tried to give each data product an image or movie where applicable and provided detailed descriptions of JSON data products. 

The page is built and viewable here: https://monodrive.readthedocs.io/en/r2v_data_products/r2v/data_processing_output/

Closes #173

### CODE REVIEW

File: docs/r2v/data_processing_output.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+<div class="img_container">
+  <video width=650px height=480px muted autoplay loop>
+    <source src="https://cdn.monodrive.io/readthedocs/redacted_vehicles.mp4" type="video/mp4">
+  </video>
+</div>
+
+## SLAM
+
+monoDrive uses Simultaneous Localization And Mapping (SLAM) in order to 
+supplment and improve the paths from the Real-to-Virtual hardware's GNSS. The
```
Comment (plarios) [inline docs/r2v/data_processing_output.md:108]: supplement

File: docs/r2v/data_processing_output.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+
+* `class_label`: The label from the segementation mask for this object
+* `bbox`: The bounding box in side of the original image
+* `mask`: THe RLE mask for the object in the image
+
+## Tracking
+
+The `tracking` directory contains all of the tracking information for recognized
+objects throughout the data run. Tracking consists of identifying objects in
+the the detected objects JSON and predicting the motion between frames. If
```
Comment (plarios) [inline docs/r2v/data_processing_output.md:192]: extra "the"

File: docs/r2v/data_processing_output.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+                "timestamp": 1593178667.093764,
+                "index": 2,
+                ...
+```
+
+For a single tracked object, the JSON defines:
+
+* `class_label`: The label from the segmentation for this type of object
+* `instance_label`: The unique label for this particular object
+* `path`: The list of frames that
```
Comment (plarios) [inline docs/r2v/data_processing_output.md:259]: incomplete sentence?

File: docs/r2v/data_processing_output.md [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+
+The mesh data generated by the Real-to-Virtual pipeline includes textured and
+untextured meshes of the different assets produced by the data processing. 
+The textured meshes that are generated from the stitched point cloud data can
+be found in the `mesh/mesh_parts` directory. These have been processed through
+several meshing algorithms for optimal triangulation for each type of object.
+
+The `mesh/tiles` directory contains the output from the monoDrive Direct Texture
+algorithm. Here, each mesh is organized into an individual tile `.obj` file that
+has been textured by project the imagery directly onto the mesh. When the
```
Comment (plarios) [inline docs/r2v/data_processing_output.md:281]: should it be "by projecting the imagery"?

File: docs/r2v/data_processing_output.md [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+
+* Configuration Files - These are text files produced by the Data Logger 
+software that contain orientation information for each sensor during parsing
+* Timestamp Files - These contain the timestamps for each data acquisition 
+iteration.
+
+## KITTI Formatted Data
+
+After parsing the monoDrive Real-to-Virtua binary data with the monoDrive 
+[mapping-toolkit](https://github.com/monoDriveIO/mapping-toolkit), the data
```
Comment (citlaligm) [inline docs/r2v/data_processing_output.md:62]: The repo is not public.

File: docs/r2v/data_processing_output.md (hunk 1 above)
Comment (bradleynull) [reply docs/r2v/data_processing_output.md:108]: Fixed

File: docs/r2v/data_processing_output.md (hunk 2 above)
Comment (bradleynull) [reply docs/r2v/data_processing_output.md:192]: Fixed

File: docs/r2v/data_processing_output.md (hunk 3 above)
Comment (bradleynull) [reply docs/r2v/data_processing_output.md:259]: Fixed

File: docs/r2v/data_processing_output.md (hunk 4 above)
Comment (bradleynull) [reply docs/r2v/data_processing_output.md:281]: Yes. Fixed.

File: docs/r2v/data_processing_output.md (hunk 5 above)
Comment (bradleynull) [reply docs/r2v/data_processing_output.md:62]: Added the caveat "(currently only available to Real-to-Virtual customers)"

**Comment by bradleynull** [comment]
@plarios @citlaligm Thanks for looking through this! I fixed all your comments.

**Pull request merged**